client/a8c-for-agencies/style.scss

```diff
@@ -136,10 +136,8 @@
 		box-sizing: border-box;
 		border-radius: 4px;
 
-		background-color: var(--color-surface);
```

Comment:
This is what I know: these styles were overriding the pressed state of the filter toggle.

Before:


https://github.com/user-attachments/assets/54a46d82-3945-4da2-9d35-d41a958acec3

After:

https://github.com/user-attachments/assets/361fa3ad-5f10-48bc-86d1-2bbdadfcf0f1

While removing them works, I'm not sure if they were targeting any other button? Not that I'm aware of but I have low confidence about this being a safe change. Would welcome advice here, @cleacos

Reply:
I did a quick review through the A4A views, and so far I found only this regression with that style in the Marketplace section:

![image](https://github.com/user-attachments/assets/9ea8fe24-b195-4278-82b5-844761097a19)

VS

![image](https://github.com/user-attachments/assets/5f84f1c8-d69b-4f7b-88d2-c4da424f00d3)

That link uses the `background-color: var(--color-surface);`